jobbergateapi2/config.py

```diff
@@ -18,7 +18,7 @@ class Settings(BaseSettings):
     SERVERLESS_STAGE: str = Field("staging")
     SERVERLESS_REGION: str = Field("eu-north-1")
     S3_BASE_PATH: str = Field("jobbergate-resources")
-    BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = []
+    BACKEND_CORS_ORIGINS: str = Field("[]")
```

Comment:
what is the reason for changing it?

Reply:
Take a look here: https://github.com/omnivector-solutions/infrastructure/pull/18

Reply:
can you add an example in anywhere? e.g. `"['https://example1.com', 'https://example2.com']"`. It seems dummy but can save some understanding in the future